install/index.md

```diff
@@ -145,14 +147,15 @@ All provisioned systems need to be RAPIDS capable. Here's what is required:
 - <i class="fas fa-check-circle"></i> [CUDA 11.8](https://developer.nvidia.com/cuda-11-8-0-download-archive){: target="_blank"} with Driver 520.61.05 or newer
 - <i class="fas fa-check-circle"></i> [CUDA 12.0](https://developer.nvidia.com/cuda-12-0-1-download-archive){: target="_blank"} with Driver 525.60.13 or newer **see CUDA 12 section below for notes on usage**
 - <i class="fas fa-check-circle"></i> [CUDA 12.2](https://developer.nvidia.com/cuda-12-2-2-download-archive){: target="_blank"} with Driver 535.86.10 or newer **see CUDA 12 section below for notes on usage**
+- <i class="fas fa-check-circle"></i> [CUDA 12.5](https://developer.nvidia.com/cuda-12-5-1-download-archive){: target="_blank"} with Driver 535.86.10 or newer **see CUDA 12 section below for notes on usage**
 
  **Note**: RAPIDS is tested with and officially supports the versions listed above. Newer CUDA and driver versions may also work with RAPIDS. See [CUDA compatibility](https://docs.nvidia.com/deploy/cuda-compatibility/index.html) for details.
 
 ## **CUDA 12 Support**
 
 ### **Docker and Conda**
 
-- <i class="fas fa-info-circle"></i> Stable CUDA 12 conda packages and Docker images currently support CUDA 12.0. Additionally, nightly versions now support CUDA 12.2
+- <i class="fas fa-info-circle"></i> Stable CUDA 12 conda packages and Docker images currently support CUDA 12.0 and 12.5.
```

Comment:
Our support for conda is not just two minor versions - we support 12.0-12.5 through CEC. We test 12.0, 12.2, and 12.5 but we know RAPIDS also works with other minor versions (because of requirements imposed by PyTorch and other libraries). This has been refined in a few places but I suppose these docs haven't been fixed yet.

We might also separate this so that Docker and conda aren't together. We only produce Docker images for specific minor versions but conda's support is broader.
```suggestion
- <i class="fas fa-info-circle"></i> Stable conda packages support CUDA 11.4-11.8 and 12.0-12.5. Docker images currently support CUDA 11.8, 12.0, and 12.5.
```